commit 48fe2ea4b60e876ddf69203c9e88b8164bba4b0a (refs/changes/44/744/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2016-10-19T16:32:41-07:00 (3 years ago)
    
    IMGAPI-595 Update node version for CVE-2016-5180
    Reviewed by: Todd Whiteman <todd.whiteman@joyent.com>
    Approved by: Todd Whiteman <todd.whiteman@joyent.com>

diff --git a/Makefile b/Makefile
index 061d9d1..b8cdf2e 100644
--- a/Makefile
+++ b/Makefile
@@ -33,7 +33,7 @@ SMF_MANIFESTS = $(shell ls smf/manifests/*.xml)
 NODEUNIT	:= ./node_modules/.bin/nodeunit
 CLEAN_FILES += ./node_modules
 
-NODE_PREBUILT_VERSION=v4.6.0
+NODE_PREBUILT_VERSION=v4.6.1
 ifeq ($(shell uname -s),SunOS)
 	NODE_PREBUILT_TAG=zone
 	# Allow building on other than image sdc-minimal-multiarch-lts@15.4.1.
